CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## v0.6.0 [2021-03-05]
+
+_What's new?_
+
+- Plugin now supports authentication through **AWS SSO**.
+- New tables added
+  - [aws_vpc_flow_log](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_vpc_flow_log)
+  - [aws_ec2_gateway_load_balancer](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_ec2_gateway_load_balancer)
```

Comment:
@LalitTurbot Can you please move the EC2 table item as the first one to preserve alphabetical order?